index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alex's Portafolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <nav class="top-menu">
+      <p class="toolbarlogo">Jhon Doe</p>
+      <img class="toolbar menu" src="./icons/Enabled.png" alt="collapsed menu icon"/>
+    </nav>
```

Comment:
[Required Changes]
- [ ]  Please place your navbar in a `<header>` tag that serves as the navigation bar section
- [ ] Also, please replace the `p` tag holding the (John Doe) logo with an `a` tag and place your name instead of the one given in the template. Except if you're John Doe 😂, you can leave it, else, please change it to your name.

Reply:
Made these changes, will keep working on the rest. Thank you Amadu!